js/packages/openinference-instrumentation-openai/src/instrumentation.ts

```diff
@@ -318,16 +331,35 @@ export class OpenAIInstrumentation extends InstrumentationBase<typeof openai> {
       },
     );
 
-    module.openInferencePatched = true;
+    try {
+      _isOpenInferencePatched = true;
```

Comment:
This line does not need to be part of the `try` clause.

Reply:
fair